build.gradle.kts

```diff
@@ -238,7 +238,7 @@ subprojects {
 
         kotlinOptions {
             allWarningsAsErrors = true
-            jvmTarget = "11"
+            jvmTarget = javaVersion.majorVersion
```

Comment:
That would mean that the JitPack packages would be published with the bytecode level of whatever version of the JVM JitPack is running, right?

Reply:
Correct, but as I've [configured JitPack to use `openjdk11`](https://github.com/oss-review-toolkit/ort/blob/16fc1c2428b15db32f8b4a0037700a2a5d95c9b2/.jitpack.yml#L2), it would still be Java 11 bytecode.